utils/check_code_style.py

```diff
@@ -239,13 +239,15 @@ def sanitize(lines, skip_checks=False):
 					i += 1
 					start_index = i + 1
 				continue
-			if (not is_string) and first_two == "//":
+			commentMatch = re.search("^//(/<?)?", line[i:i + 4])
+			if (not is_string and not commentMatch is None):
```

Comment:
```suggestion
			if (not is_string and commentMatch is not None):
```
I think that's the convention in Python. Though you should be able to just use the truthy value of the object itself, as we don't have to worry about it being empty:
```suggestion
			if (not is_string and commentMatch):
```